assets/js/components/dashboard-sharing/DashboardSharingSettingsButton.js

```diff
@@ -87,6 +92,13 @@ export default function DashboardSharingSettingsButton() {
 					open={ dialogOpen }
 					onClose={ closeDialog }
 					className="googlesitekit-dialog googlesitekit-sharing-settings-dialog"
+					// On mobile, the dialog box's flexbox is set to stretch items within to cover
+					// the whole screen. But we have to move the box and adjust its height below the
+					// WP Admin bar of 46px which gradually scrolls off the screen.
+					style={ {
```

Comment:
@jimmymadon Nice! I'am thinking maybe we should not set the styles dynamically here on larger breakpoints, i.e we just set them on small breakpoints. we can use the existing `breakpoint` value we have to decide whether or not to add those styles.